Cargo.toml

```diff
@@ -107,10 +108,13 @@ members = ["crates/*", "sdk/rust", "sdk/rust/macro"]
 
 [workspace.dependencies]
 tracing = { version = "0.1", features = ["log"] }
-wasmtime-wasi = { version = "13.0.0", features = ["tokio"] }
-wasi-common-preview1 = { version = "13.0.0", package = "wasi-common" }
-wasmtime = { version = "13.0.0", features = ["component-model"] }
-spin-componentize = { git = "https://github.com/fermyon/spin-componentize", rev = "f29cdb26a6b1700ae1fe48c94bbef26a8069d566" }
+wasmtime-wasi-http = { git = "https://github.com/bytecodealliance/wasmtime", rev = "17505667fa2895b3bd31bb9f5f4c544250090926" }
```

Comment:
Using a snapshot of Wasmtime 14 until it is released, since that will be the earliest version that supports WIT resources and inbound `wasi-http`.

Reply:
Should we switch to using [the release branch](https://github.com/bytecodealliance/wasmtime/tree/release-14.0.0)?